### DISCUSSION TITLE
CORS vercel

### DISCUSSION THREAD

**Discussion by reyptr27** [dewanakl/undangan#34] [General]
kapan hari sukses deploy dan fungsi berjalan dengan lancar tanpa error mas
tapi setelah seminggu kok muncul error begini yaa ??

has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

btw front end saya deploy di netlify
dan api / back end saya deploy di vercel

tolong dibantu mas 😁

**Comment by dewanakl** [comment]
Boleh coba lampirkan ss consolenya mas?

**Reply by reyptr27** [reply]
![Screenshot 2024-08-06 100251](https://github.com/user-attachments/assets/2505289f-67ba-438a-a8eb-cc4b96f4da58)

ini mas ss nya, tolong mas saya sudah cari - cari di google tapi tidak menemukan solusinya.

**Reply by dewanakl** [reply]
Coba cek db supabase nya mas, biasanya kalau jarang digunakan akan masuk mode sleep.

**Reply by reyptr27** [reply]
aman kok mas db nya, sepertinya request ke api nya

![Screenshot 2024-08-06 111512](https://github.com/user-attachments/assets/701a97d1-576f-4bff-818b-3d37353dd3bd)

![Screenshot 2024-08-06 111534](https://github.com/user-attachments/assets/9343f5cc-4952-4afb-a5bc-3f1e04452d36)

**Reply by dewanakl** [reply]
bisa coba melalui postman dulu mas

**Reply by reyptr27** [reply]
seperti ini mas ??

![Screenshot 2024-08-06 112815](https://github.com/user-attachments/assets/17489e06-4bee-4343-89ac-3468a5ff867c)


response nya kok 200 ya ?

**Reply by dewanakl** [reply]
coba cek kembali file vercel.json nya mas, dan lakukan redeploy juga

**Reply by reyptr27** [reply]
sudah redeploy tetap gabisa mas

**Comment by reyptr27** [comment]
kalo begini gimana ya mas ??

![Screenshot 2024-08-06 142447](https://github.com/user-attachments/assets/002a0bf2-bcc3-40c7-90d1-610fe90faafe)

**Reply by dewanakl** [reply]
Sudah bisa mas

**Reply by reyptr27** [reply]
untuk data-key di index.html dan dashboard.html itu diisi apa sebenarnya mas ?

**Reply by dewanakl** [reply]
Bisa lihat di slide nya mas
